src/intervals/functions.jl

```diff
@@ -4,61 +4,140 @@
 # Integer power:
 function ^{T<:Real}(a::Interval{T}, n::Integer)
```

Comment:
OK, if it's passing the ITF1788 tests then that's a pretty good indication that it's correct!
Let's leave it for now.

By the way, I opened issue #65 about the need to incoporate the ITF1788 tests directly as part of our test suite somehow. This could be by making the ITF1788 suite a (test) dependency, for example. But that may be painful for Travis. Can we just copy over the files that ITF1788 outputs and add them to our test collection? (And then update when necessary.)